proposals/build-cli-options.md

```diff
@@ -0,0 +1,50 @@
+# [Proposal] Allow additional cli options to pass to the build command
+
+Related to:
+- https://github.com/devcontainers/spec/discussions/301
+- https://github.com/devcontainers/spec/pull/324
+- https://github.com/microsoft/vscode-remote-release/issues/3545
+
+## Goal
+
+Extend the build section of the configuration with an option to pass additional arguments via cli.
+
+## Proposal
+
+Introduce the property `cliOptions` to the `buildOptions` in the schema. It is very similar to `runArgs`, which contains cli-arguments passed to the `run` command, the `cliOptions` passes cli-arguments to the `build` command.
+
```

Comment:
Can we also add a line in the proposal stating that the new property will be of type `Array (string)`

Reply:
I adjusted the text a bit and also mention the type.